### ISSUE TITLE
WARNING: indentation of => is not properly aligned

### ISSUE THREAD

**Issue by jeff1evesque** [rodjek/puppet-lint#447]
The following manifest:

```puppet
## define $PATH for all execs, and packages
Exec {path => ['/usr/bin/']}

## mysql::server: install, and configure mariadb-server
#
#  @password_hash, default password (can be adjusted via cli)
#  @max_connections_per_hour, @max_queries_per_hour, @max_updates_per_hour,
#      @max_user_connections, a zero value indicates no limit
class {'::mysql::server':
  package_name => 'mariadb-server',
  root_password => 'password',
  users => {
    'authenticated@localhost' => {
      ensure => 'present',
      max_connections_per_hour => '0',
      max_queries_per_hour => '0',
      max_updates_per_hour => '0',
      max_user_connections => '0',
      password_hash        => mysql_password('password'),
    },
    'provisioner@localhost' => {
      ensure => 'present',
      max_connections_per_hour => '1',
      max_queries_per_hour => '0',
      max_updates_per_hour => '0',
      max_user_connections => '1',
      password_hash        => mysql_password('password'),
    },
  },
  grants => {
    'authenticated@localhost/db_machine_learning.*' => {
      ensure => 'present',
      options => ['GRANT'],
      privileges => ['INSERT', 'DELETE', 'UPDATE', 'SELECT'],
      table => 'db_machine_learning.*',
      user => 'authenticated@localhost',
    },
    'provisioner@localhost/db_machine_learning.*' => {
      ensure => 'present',
      options => ['GRANT'],
      privileges => ['CREATE'],
      table => 'db_machine_learning.*',
      user => 'provisioner@localhost',
    },
  },
  databases => {
    'db_machine_learning' => {
      ensure => 'present',
      charset => 'utf8',
    },
  }
}

## mysql::client: install, and configure mariadb-client
class {'::mysql::client':
  package_name => 'mariadb-client',
  require => Class['::mysql::server'],
}

## mysql::bindings: install python-mariadb bindings
class {'::mysql::bindings':
  python_enable => 'true',
  require => [Class['::mysql::client'], Class['::mysql::server']],
}

## define database tables
#
#  @require, syntax involves 'Class Containment'. For more information,
#      https://puppetlabs.com/blog/class-containment-puppet
exec {'create-database-tables':
  command => 'python setup_tables.py',
  cwd => '/vagrant/puppet/scripts/',
  require => Class['::mysql::bindings::python'],
}
```

generates the following error:

```bash
puppet/manifests/setup_database.pp - WARNING: quoted boolean value found on line 62
puppet/manifests/setup_database.pp - WARNING: indentation of => is not properly aligned on line 14
puppet/manifests/setup_database.pp - WARNING: indentation of => is not properly aligned on line 16
puppet/manifests/setup_database.pp - WARNING: indentation of => is not properly aligned on line 17
puppet/manifests/setup_database.pp - WARNING: indentation of => is not properly aligned on line 18
puppet/manifests/setup_database.pp - WARNING: indentation of => is not properly aligned on line 19
puppet/manifests/setup_database.pp - WARNING: indentation of => is not properly aligned on line 22
puppet/manifests/setup_database.pp - WARNING: indentation of => is not properly aligned on line 24
puppet/manifests/setup_database.pp - WARNING: indentation of => is not properly aligned on line 25
puppet/manifests/setup_database.pp - WARNING: indentation of => is not properly aligned on line 26
puppet/manifests/setup_database.pp - WARNING: indentation of => is not properly aligned on line 27
puppet/manifests/setup_database.pp - WARNING: indentation of => is not properly aligned on line 2
puppet/manifests/setup_database.pp - WARNING: indentation of => is not properly aligned on line 32
puppet/manifests/setup_database.pp - WARNING: indentation of => is not properly aligned on line 33
puppet/manifests/setup_database.pp - WARNING: indentation of => is not properly aligned on line 35
puppet/manifests/setup_database.pp - WARNING: indentation of => is not properly aligned on line 36
puppet/manifests/setup_database.pp - WARNING: indentation of => is not properly aligned on line 39
puppet/manifests/setup_database.pp - WARNING: indentation of => is not properly aligned on line 40
puppet/manifests/setup_database.pp - WARNING: indentation of => is not properly aligned on line 42
puppet/manifests/setup_database.pp - WARNING: indentation of => is not properly aligned on line 43
puppet/manifests/setup_database.pp - WARNING: indentation of => is not properly aligned on line 38
puppet/manifests/setup_database.pp - WARNING: indentation of => is not properly aligned on line 48
puppet/manifests/setup_database.pp - WARNING: indentation of => is not properly aligned on line 10
puppet/manifests/setup_database.pp - WARNING: indentation of => is not properly aligned on line 12
puppet/manifests/setup_database.pp - WARNING: indentation of => is not properly aligned on line 30
puppet/manifests/setup_database.pp - WARNING: indentation of => is not properly aligned on line 46
puppet/manifests/setup_database.pp - WARNING: indentation of => is not properly aligned on line 57
puppet/manifests/setup_database.pp - WARNING: indentation of => is not properly aligned on line 63
puppet/manifests/setup_database.pp - WARNING: indentation of => is not properly aligned on line 72
puppet/manifests/setup_database.pp - WARNING: ensure found on line but it's not the first attribute on line 14
```

This is confusing, and frustrating.  It almost seems more beneficial to not use this puppet-lint project, as the linting creates unnecessary trivial-busy work, that seems convoluted.  I have to figure out why the correct syntax that works, and properly indented, is giving warning suggesting the indentation is off.

**Comment by jeff1evesque** [comment]
From a quick pass through of #446, I get the impression that I have to rewrite the working valid puppet syntax in the corresponding manifest, simply to make the linting service agree.

**Comment by jeff1evesque** [comment]
The above confusion is actually not the fault of this project.  It's regulated by puppet [arrow-alignment](https://docs.puppetlabs.com/guides/style_guide.html#arrow-alignment) styleguide.
